ingestion-sink/src/main/java/com/mozilla/telemetry/ingestion/sink/config/SinkConfig.java

```diff
@@ -298,9 +302,21 @@ final long getMaxOutstandingRequestBytes(Env env) {
     }
   }
 
-  private static PubsubMessageToObjectNode.Format getFormat(Env env) {
-    return PubsubMessageToObjectNode.Format.valueOf(
-        env.getString(OUTPUT_FORMAT, PubsubMessageToObjectNode.Format.RAW.name()).toUpperCase());
+  private static PubsubMessageToObjectNode getFormat(Env env) {
+    final String format = env.getString(OUTPUT_FORMAT, "raw").toLowerCase();
+    switch (format) {
+      case "raw":
+        return PubsubMessageToObjectNode.Raw.of();
+      case "decoded":
+        return PubsubMessageToObjectNode.Decoded.of();
+      case "payload":
+        // TODO
+        return PubsubMessageToObjectNode.Payload.of(env.getStrings(STRICT_SCHEMA_DOCTYPES, null),
+            env.getString(SCHEMAS_LOCATION), env.getString(SCHEMA_ALIASES_LOCATION, null),
```

Comment:
@whd for [bug 1612367](https://bugzilla.mozilla.org/show_bug.cgi?id=1612367) would you please deploy this branch as a nonprod live sink, with `STRICT_SCHEMA_DOCTYPES` as an optional CSV of `namespace/doctype` values, `SCHEMAS_LOCATION` as a local path to `schemas.tar.gz`, `SCHEMA_ALIASES_LOCATION` as an optional local path to an aliasing config, and `OUTPUT_FORMAT=payload`.

Reply:
Yep. I'll ping you if I have trouble setting this up. Do you want the 100% tests (i.e. different temp tables and prod subscriptions) or just the nonprod sample tables?

Reply:
let's go with 100%, and `OUTPUT_MODE=mixed`

Reply:
See https://github.com/mozilla/gcp-ingestion/issues/1136; I believe there's no point in supporting schema aliases.